util/src/main/java/tc/oc/pgm/util/tablist/TabView.java

```diff
@@ -229,21 +230,14 @@ private void renderLegacy() {
           if (this.slots[i] == this.rendered[i] && !this.slots[i].isDirty(this)) continue;
           this.rendered[i] = this.slots[i];
 
-          this.display.set(x, y, toPlain(this.rendered[i].getContent(this)));
+          this.display.set(
+              x, y, TextTranslations.translateLegacy(this.rendered[i].getComponent(this), viewer));
```

Comment:
See the coment above regarding this, it is bound to break from differing implementations.

Reply:
This cuts down on unneeded serialization and deserialization.

Reply:
It does, but it's wrong. Editing the result of BaseComponent[] does not reflect here, and it's something used by extending implementations

Reply:
I see your point